sig-docs/blog-subproject/README.md

```diff
@@ -22,73 +22,4 @@ Regular Blog Meeting: Tuesdays at 18:30 UTC (biweekly). [Convert Your Timezone](
 - Open Community Issues/PRs: [`is:open repo:kubernetes/website label:area/blog`](https://github.com/issues?q=is%3Aopen+label%3Aarea%2Fblog+repo%3Akubernetes%2Fwebsite)
 - GitHub Teams: [@kubernetes/kubernetes-blog](https://github.com/orgs/kubernetes/teams/kubernetes-blog), [@kubernetes/kubernetes-blog-maintainers](https://github.com/orgs/kubernetes/teams/kubernetes-blog-maintainers)
 
-## Submit a Post
-
-Anyone can write a blog post and submit it for review. Blog posts should not be commercial in nature and should consist of content that will apply broadly to the Kubernetes community.
-
-To submit a blog post, follow the steps below.
-
-1. [Sign the CLA](https://kubernetes.io/docs/contribute/start/#sign-the-cla) if you have not yet done so.
-1. Have a look at the Markdown format for existing blog posts in the [website repository](https://github.com/kubernetes/website/tree/master/content/en/blog/_posts).
-1. Write out your blog post in a text editor of your choice.
-1. On the same link from step 2, click the Create new file button. Paste your content into the editor. Name the file to match the proposed title of the blog post, but don’t put the date in the file name. The blog reviewers will work with you on the final file name and the date the blog will be published.
-1. When you save the file, GitHub will walk you through the pull request process.
-1. A blog post reviewer will review your submission and work with you on feedback and final details. When the blog post is approved, the blog will be scheduled for publication.
-
-### Blog Guidelines
-
-Requested Content (with examples):
-
-- New Kubernetes capabilities
-- Kubernetes projects updates
-- Updates from Special Interest Groups
-- Tutorials and walkthroughs
-- Thought leadership around Kubernetes
-- Kubernetes Partner OSS integration
-- **Original content only**
-
-Unsuitable Content:
-
-- Vendor product pitches
-- Partner updates without an integration and customer story
-- Syndicated posts (language translations ok)
```

Comment:
I'd like to incorporate these 2 lists into the k/website page about blog contributions.